source/gui/addonGui.py

```diff
@@ -248,8 +248,9 @@ def __init__(self, parent):
 		self.getAddonsButton = generalActions.addButton(self, label=_("&Get add-ons..."))

```

Comment:
This button should also be disabled when in secure mode since it opens a web browser from which you can easily do a lot of insecure stuff.